api/cds.proto

```diff
@@ -148,11 +176,9 @@ message Cluster {
   // rate. If this setting is not specified, the value defaults to 5000. For
```

Comment:
Not your change, but I would s/dns/DNS/ in this block (and add links to the enum values).